### ISSUE TITLE
[Feature]: Line chart with config item on x-axis, multiple runs per line

### ISSUE THREAD

**Issue by rubencart** [wandb/wandb#4658]
### Description

I have several finetuning runs evaluating checkpoints of other pretraining runs. Pretraining run A creates checkpoints A-1 and A-2 in epochs 1 and 2. Now I have a finetuning run for each checkpoint of A. I would like to create a chart with on the X-axis the pretraining epoch (I can get this from a config file), not the finetuning epoch, and on the Y-axis a test metric I compute once at the end of every finetuning run. The above is easy to get with a scatter plot, but I would like to be able to group finetuning runs by the pretraining run (so A, that has a number of separate finetuning runs associated with it, not the pretraining checkpoint A-1 for which there is 1 finetuning run)(I have this info also in a config file), so that each line in the resulting line chart has points from different finetuning runs corresponding to different epochs of the same pretraining runs. Is that possible, to group runs in a line chart?

### Suggested Solution

Be able to group runs according to a config setting, in a line chart, so that one line can consist of metrics of several runs.

**Comment by thanos-wandb** [comment]
Hi @rubencart thanks a lot for posting this question and describing your use case. Since this is a duplicate request, I will close the GitHub ticket here and we can continue the discussion in our [Forum](https://community.wandb.ai/t/line-chart-with-config-item-on-x-axis-multiple-runs-per-line/3548).
